widgetastic_patternfly.py

```diff
@@ -156,6 +156,10 @@ def assert_message(self, text, t=None):
                 e_text = text
             raise AssertionError('assert_message: {}'.format(e_text))
 
+    def assert_success_message(self, text, t=None):
```

Comment:
well, the message type is given by the function name - assert_success_message.

I think you added this because of my comments about assert_success_message() - That was my fault for thinking you were calling assert_message() from web_ui.flash.

assert_message() here takes a type, so in your use of it you could have supplied the type to get the same level of verification (success type message + text).

I'm not sure assert_success_message() should check assert_no_error(). There could be multiple messages, and we're only asserting that one of those messages is a success type message matching the given text.

Really it boils down to this - I think assert_success_message should be a wrapper for assert_message where t='success' by default.